src/cont.rs

```diff
@@ -12,13 +12,13 @@ pub enum Continuation<F: LurkField> {
         continuation: ContPtr<F>,
     },
     Call {
-        unevaled_arg: Ptr<F>,
         saved_env: Ptr<F>,
+        unevaled_arg: Ptr<F>,
```

Comment:
Was there a reason to reorder the fields here and elsewhere in this file? If not, let's minimize the changeset.